### PR TITLE
chore(server): add more tests

### PR DESCRIPTION
**Pull request by shaojunda**
1. add some tests.
2. fix some bug.

### CODE REVIEW

File: server/app/models/cell_output.rb [hunk 1]
```diff
@@ -17,7 +17,7 @@ def to_node_cell_output
       capacity: capacity,
       data: data,
       lock: lock_script.to_node_lock,
-      type: type_script.try(:to_node_lock)
+      type: type_script.try(:to_node_type)
```
Comment (classicalliu) [inline server/app/models/cell_output.rb]: `try` or `try!`?
```suggestion
      type: type_script&.to_node_type
```

File: server/test/factories/cell_output.rb [hunk 2]
```diff
@@ -1,7 +1,7 @@
 FactoryBot.define do
   factory :cell_output do
     capacity { 50000 }
-    data {}
+    data { [] }
```
Comment (classicalliu) [inline server/test/factories/cell_output.rb]: Why data is an array ?

File: server/test/models/ckb_sync/persist_test.rb [hunk 3]
```diff
@@ -291,6 +291,27 @@ class PersistTest < ActiveSupport::TestCase
...
...00"},"timestamp":"1555604163266","txs_commit":"0x9defbef60635e92d77ec14a393e0e9701f87b02190bf3bbb37be760946ac4f73","txs_proposal":"0x0000000000000000000000000000000000000000000000000000000000000000","uncles_count":0,"uncles_hash":"0x0000000000000000000000000000000000000000000000000000000000000000","version":0,"witnesses_root":"0x0000000000000000000000000000000000000000000000000000000000000000"},"proposal_transactions":[]},{"header":{"difficulty":"0x1000","hash":"0x6af4cb1d4b2f8d6b05be9a6d713203ae9f3191b2cab805fe1ebeec12448e737a","number":"1","parent_hash":"0x298f349c8cdfadf46e8008e72afe6da78b1ea1b7d86470ea71bb0e404c5c9d7f","seal":{"nonce":"7551133712902986728","proof":"0x81070000841f0000f7210000a022000037230000d22f00003c4900003c5300000d5a00000d640000c46d00004f7c0000"},"timestamp":"1555604128584","txs_commit":"0xbd9ed8dec5288bdeb2ebbcc4c118a8adb6baab07a44ea79843255ccda6c57915","txs_proposal":"0x0000000000000000000000000000000000000000000000000000000000000000","uncles_count":0,"uncles_hash":"0x0000000000000000000000000000000000000000000000000000000000000000","version":0,"witnesses_root":"0x0000000000000000000000000000000000000000000000000000000000000000"},"proposal_transactions":[]}]
```
Comment (classicalliu) [inline server/test/models/ckb_sync/persist_test.rb]: How long the line is ....

File: server/test/factories/cell_output.rb (hunk 2 above)
Comment (shaojunda) [reply server/test/factories/cell_output.rb]: should not, I will fix this.

File: server/test/models/ckb_sync/persist_test.rb (hunk 3 above)
Comment (shaojunda) [reply server/test/models/ckb_sync/persist_test.rb]: very, but it's fake data, need to adjust?

File: server/app/models/cell_output.rb (hunk 1 above)
Comment (shaojunda) [reply server/app/models/cell_output.rb]: ok, I got it. thank you.

File: server/test/models/ckb_sync/persist_test.rb (hunk 3 above)
Comment (classicalliu) [reply server/test/models/ckb_sync/persist_test.rb]: let it go

**Pull request merged**